Tests/Unit/Admin/Extension/PublishTimePeriodExtensionTest.php

```diff
@@ -34,6 +36,26 @@ public function testFormMapper()
             ->method('add')
             ->will($this->returnSelf());
 
+        $builder = $this->getMockBuilder(FormBuilderInterface::class)->getMock();
+        $builder->expects($this->exactly(2))
+            ->method('get')
+            ->withConsecutive(['publish_start_date'], ['publish_end_date'])
+        ;
+        $formFactory = $this->getMockBuilder(FormFactoryInterface::class)->getMock();
+        $formFactory->expects($this->once())
+            ->method('createBuilder')
+            ->will($this->returnValue($builder))
+        ;
```

Comment:
Thanks for the advice. However this is no longer relevant as I removed the builder stuff from extensions.